### ISSUE TITLE
No license file

### ISSUE THREAD

**Issue by SoarinFerret** [VicentGJ/AD-webmanager#108]
This project currently has no license. Without a license, there are no assumptions that can be made about what anyone is allowed to do with the code. Is there a chance a license could be applied to this project?

**Comment by csutcliff** [comment]
bottom line of the readme:

> The project is under the MIT license.

good enough for me.

**Comment by SoarinFerret** [comment]
Is it allowed to be MIT? The code it is forked from is under a GPL 2 license.

**Comment by SoarinFerret** [comment]
But I did fail to notice that, so thanks for pointing that out!

**Comment by VicentGJ** [comment]
Yes you are absolutely right, we thought at first that the huge amount of changes we did to the original granted as permission to change the license. I was wrong,  but even after that we didn't change it since we planned to release a full rework of the project using fastapi, sveltekit and an entirely new codebade. Since that is not coming along as soon as we planned, I probably should address this issue now.
Sorry for the long story but I felt the need to explain the confusion regarding the license 😅.
Thanks for the heads up!
